Automattic/wp-calypso
# Add woocommerce fallback menu item + fix jetpack icon

#### Changes proposed in this Pull Request

* Always show woocommerce as a fallback menu item when the `woop` flag is enabled regardless of plan level.
* Fix the jetpack icon (was previously using a blank square - the image mask copy pasted from somewhere else by the looks)

#### Testing instructions

The easiest way to test this is to [change this line](https://github.com/Automattic/wp-calypso/blob/0b1198ce5237a2540aaeeff024da4d5532a84acb/client/my-sites/sidebar-unified/use-site-menu-items.js#L81) to:
```
return buildFallbackResponse( fallbackDataOverides );
```

The fallback menu is only shown while the menu is loaded from the masterbar code in jetpack/public-api (D67193-code) or if there is a network error. 

To test, you need to watch the menu clearly and avoid caching by switching to a site you haven't loaded in a while.

E.g. start here: http://calypso.localhost:3000/home/wooptestnowoo.wordpress.com?flags=woop

Then switch to a different site and observe the WooCommerce menu item before it gets removed when the menu finishes loading.

Then test without the flag (Default mode): http://calypso.localhost:3000/home/wooptestnowoo.wordpress.com?flags=-woop

The WooCommerce menu item won't show during fallback unless it's a site with business + atomic + no woo install.

#### Before:
![Screenshot 2021-09-23 at 16-51-45 My Home ‹ wooptest-nowoo — WordPress com](https://user-images.githubusercontent.com/811776/134465396-cb73ef97-5735-4601-a7f4-aa16631dc337.png)

#### After:
![Screenshot 2021-09-23 at 16-50-57 My Home ‹ wooptest-nowoo — WordPress com](https://user-images.githubusercontent.com/811776/134465341-146556f6-03e1-45df-ac83-9101c0000c91.png)


Related to #55428
